nb

```diff
@@ -440,8 +440,15 @@ _option_value_is_present() {
 #
 # Default: `$HOME/.nbrc`
 #
-# The location of the .nbrc configuration file.
-export NBRC_PATH="${NBRC_PATH:-"${HOME}/.${_ME}rc"}"
+# Check if XDG_CONFIG_HOME exists
+if [[ -n "${XDG_CONFIG_HOME}" ]]; then
+  # If XDG_CONFIG_HOME exists put .nbrc config file in a subdirectory.
+  mkdir -p "${XDG_CONFIG_HOME}/${_ME%%.*}" &&
+  export NBRC_PATH="${NBRC_PATH:-"${XDG_CONFIG_HOME}/${_ME%%.*}/.${_ME}rc"}"
```

Comment:
In my opinion, there is no need for the nbrc file to be a dotfile if it is under `XDG_CONFIG_HOME/nb` already.
```suggestion
  export NBRC_PATH="${NBRC_PATH:-"${XDG_CONFIG_HOME}/${_ME%%.*}/${_ME}rc"}"
```